code/modules/food_and_drink/plants.dm

```diff
@@ -872,6 +884,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 	icon_state = "poison"
 	food_color = "#AC1515"
 	initial_volume = 100
+	brew_result = "capulettium"
```

Comment:
This ensures 20 units instead of engaging with potency system, which seems like a cleaner means of developing the poison.